networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/Eth2PeerManager.java

```diff
@@ -98,7 +100,12 @@ public void onConnect(final Peer peer) {
           .sendStatus()
           .finish(
               () -> LOG.trace("Sent status to {}", peer.getId()),
-              (err) -> LOG.debug("Failed to send status to {}: {}", peer.getId(), err));
+              RootCauseExceptionHandler.builder()
+                  .addCatch(
+                      RpcException.class,
+                      err -> LOG.trace("Status message rejected by {}: {}", peer.getId(), err))
+                  .defaultCatch(
+                      err -> LOG.debug("Failed to send status to {}: {}", peer.getId(), err)));
```

Comment:
I got sick of the usual:
```
err -> {
  if (Throwables.getRootCause(err) instanceof RpcException) {
     // do one thing
  } else {
    // do something else
  });
```

I think this is slightly better but it's not as succinct as I was hoping for.